### PR TITLE
Prepare 2.8.2

### PR DESCRIPTION
**Pull request by leplatrem**
**Tooling requirements**
* ``pip install zest.releaser[recommended]`` (Install zest.releaser, wheel and twine)

**Step 1**
```
$ git co -b prepare-X.Y.Z
$ prerelease
$ vim docs/conf.py
$ rm -fr .venv/
$ make install
$ pip freeze > requirements.txt
$ # Remove the master URL in the requirements.txt file.
$ git ci -a --amend
$ git push origin prepare-X.Y.Z
```
* [x] Update Contributors file ``git shortlog -sne | awk '{$1=""; sub(" ", ""); print}' | awk -F'<' '!x[$1]++' | awk -F'<' '!x[$2]++' | sort``
* [x] Merge remaining pull requests
* [x] Update changelog
* [x] Update version in ``cliquet_docs/conf.py``
* [x] Update known good set of dependencies in requirements.txt

**Step 2**
```
$ git co master
$ git merge --no-ff prepare-X.Y.Z
$ release
$ postrelease
```
* [ ] Tag vX.Y.Z
* [ ] Publish on Pypi

**Step 3**
* [ ] Close milestone in Github
* [ ] Create next milestone in Github
* [ ] Add entry in Github release page
* [ ] Configure the version in ReadTheDocs
* [ ] Send mail to ML

### CODE REVIEW

**Comment by Natim** [comment]
Looks good to me.

File: requirements.txt [hunk 1]
```diff
@@ -8,19 +8,18 @@ psycopg2==2.6.1
...
 six==1.10.0
-statsd==3.1
+statsd==3.2
 structlog==15.3.0
 translationstring==1.3
 ujson==1.33
 venusian==1.0
 WebOb==1.5.0
 Werkzeug==0.10.4
-wheel==0.24.0
```
Comment (Natim) [inline requirements.txt]: Did you remove it by hand or is it your pip freeze that is missing it?

**Comment by Natim** [comment]
Please rebase and then we can merge.

File: requirements.txt (hunk 1 above)
Comment (leplatrem) [inline requirements.txt]: I did not remove it by hand :/

**Comment by leplatrem** [comment]
> Please rebase and then we can merge.

I won't rebase,  the master contains changes that we don't want in this minor release :)

File: requirements.txt (hunk 1 above)
Comment (Natim) [inline requirements.txt]: Ok perfect then :+1:

**Comment by Natim** [comment]
> I won't rebase, the master contains changes that we don't want in this minor release :)

Ok, then we probably want to create a ``2.8.x`` branch and then merge it into master.

**Pull request merged**